src/ray/raylet/scheduling_queue.cc

```diff
@@ -45,28 +45,72 @@ namespace ray {
 
 namespace raylet {
 
+SchedulingQueue::TaskQueue::~TaskQueue() {
+  task_map_.clear();
+  task_list_.clear();
+}
+
+bool SchedulingQueue::TaskQueue::AppendTask(const TaskID &task_id, const Task &task) {
+  RAY_CHECK(task_map_.find(task_id) == task_map_.end());
+  auto list_iter = task_list_.insert(task_list_.end(), task);
```

Comment:
I know it's annoying, but the style policy for Ray is to have full, descriptive variable names. In this case `list_iterator` and `iterator` instead of `list_iter` and `iter`.

Reply:
update. thanks.